snowflake-cortex/app/manifest.yml

```diff
@@ -0,0 +1,17 @@
+# For more information on creating manifest, go to https://docs.snowflake.com/en/developer-guide/native-apps/creating-manifest
+manifest_version: 1
+
+version:
+  name: Dev
+  label: "Dev Version"
+  comment: "Default version used for development. Override for actual deployment."
```

Comment:
I want to start removing the version from `manifest.yml` as the best practice is to provide these values via `snow app version create`.

Reply:
makes sense!